scripts/image-builder/config/kickstart.ks.template

```diff
@@ -37,10 +37,6 @@ ostreesetup --nogpg --osname=rhel --remote=edge --url=file:///run/install/repo/o
 
 %post --log=/var/log/anaconda/post-install.log --erroronfail
 
-# Update the ostree server URL
-ostree remote delete edge
-ostree remote add --no-gpg-verify edge REPLACE_OSTREE_SERVER_URL
```

Comment:
The ostreesetup command uses the `file:///run/install/repo/ostree/repo` path, which is necessary during the installation. In the post-install stage, we're replacing it by REPLACE_OSTREE_SERVER_URL.
I do not understand why this replacement is not necessary - where is  REPLACE_OSTREE_SERVER_URL set?

Reply:
I keep forgetting this file is used in a different way. I've removed this change from the PR.